packages/mds-stream/index.ts

```diff
@@ -135,15 +168,25 @@ async function writeStreamBatch(stream: Stream, field: string, values: unknown[]
 
 // put basics of vehicle in the cache
 async function writeDevice(device: Device) {
+  if (env.SINK) {
+    return write('mds.device', JSON.stringify(device))
```

Comment:
I think `write` is a pretty generic name here. Should we use `emitCloudEvent` or something more narrative, similar to how we call it `writeStream`?